e2e/samplesMarkup.e2e-spec.ts

```diff
@@ -30,6 +30,10 @@ test.describe.parallel(`Samples markup consistency check`, () => {
 			if (route === 'accordion/playground' && testInfo.project.name.startsWith('angular:')) {
 				test.skip();
 			}
+			// skipping consistency checks for react and svelte for slider
+			if (testInfo.project.name.startsWith('react:') || testInfo.project.name.startsWith('svelte:')) {
```

Comment:
Where do you check that it is for the slider? Don't disable samples markup check in react and svelte for all widgets!

Reply:
oh, missed the line with the route :D